packages/ceramic-cli/src/ceramic-daemon.ts

```diff
@@ -21,6 +22,16 @@ interface CreateOpts {
   anchorServiceUrl?: string;
 }
 
+function logErrors (err: Error, req: Request, res: Response, next: NextFunction) {
+  console.error(err)
+  next(err)
+}
```

Comment:
Should we really modify the behavior of the daemon if we log the errors? Why not both log the error and send the response?

Reply:
This does both (if `DEBUG` is set) - when we call `next(err)` at the end, it gets passed to the next handler, in the order they are registered.